### PR TITLE
Image CSS

### PR DESCRIPTION
**Pull request by Matt-Butler**
Adding CSS for image component. 
Adding project structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
-# Terra Image
+# Terra Image _(Currently in beta)_
```
Comment (gneatgeek) [inline README.md]: Do we need to have the beta notice since we'll be releasing this as 1.0.0

File: tests/visual/terra-image-circle.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<img src="https://placeholdit.imgix.net/~text?txtsize=30&txt=320%C3%97320&w=320&h=320" alt="circle" class="terra-Image terra-Image--circle">
```
Comment (gneatgeek) [inline tests/visual/terra-image-circle.html]: but kittens... https://placekitten.com/

File: tests/visual/terra-image-grid.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<div class="terra-Grid terra-Grid--auto">
+  <div class="terra-Grid-col">
+    <img src="https://placeholdit.imgix.net/~text?txtsize=30&txt=320%C3%97320&w=320&h=320" alt="circle" class="terra-Image terra-Image--circle terra-Image--fluid">
```
Comment (gneatgeek) [inline tests/visual/terra-image-grid.html]: Can the alt text be more verbose.  e.g. "example of circle image styles"

File: docs/terra-image.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Terra Image _(Currently in beta)_
```
Comment (bjankord) [inline docs/terra-image.md]: Do we need to have the beta notice since we'll be releasing this as 1.0.0

File: package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-image",
-  "version": "0.0.0",
-  "description": "Under construction",
+  "version": "0.1.0",
```
Comment (bjankord) [inline package.json]: Should this be 1.0.0

Comment (bjankord) [inline package.json]: Cool 👍

**Pull request merged**